src/web/routes/application/children-dob/handle-add-action.js

```diff
@@ -0,0 +1,16 @@
+const { PATH } = require('./constants')
+
+const addActionRequested = body => body.hasOwnProperty('add')
+
+const handleAddAction = (req, res, next) => {
+  if (!addActionRequested(req.body)) {
```

Comment:
Can we change this so the if statement is a positive rather than a negative. i.e. `if (addActionRequested(req.body))`. Reads much nicer that way.

Reply:
Presumably that would look like this:
```
const handleAddAction = (req, res, next) => {
  if (addActionRequested(req.body)) {
    req.session.children.inputCount = req.session.children.childCount + 1
    return res.redirect(PATH)
  }
  return next()
}
```

Reply:
I normally prefer it the way it currently is, have the alternate flow to return/error quickly and then have the main body of the code to be for the positive case that the method was written for rather than be be completely within an if block.

Reply:
If we followed the suggested approach in the remove middleware we would end up with nested if blocks, and it makes sense that they follow the same format. Anyone feel like voting on this??

Reply:
This also means that all logic for the intended function would need to be inside an `if` block .. on second thoughts I'm not sure I agree with the suggestion, although I get the point about how it reads.

Reply:
For the remove middleware I'd leave it as is which is why I didn't leave a comment there.

I'd argue readability is more important, not to mention the `if(!condition) { next() }` format isn't consistent with existing middleware.

Reply:
I accept this is very much a preference matter so don't really mind if you prefer the way it is.

Reply:
If we're voting, I'd leave it as it is, I don't agree with Dave here I'm afraid.

Reply:
@dwybourn if you're not too fussed then I'm going to leave this as is.